test/es6/module-namespace.js

```diff
@@ -0,0 +1,97 @@
+//-------------------------------------------------------------------------------------------------------
+// Copyright (C) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+//-------------------------------------------------------------------------------------------------------
+
+// ES6 Module syntax tests -- verifies syntax of import and export statements
+
+WScript.LoadScriptFile("..\\UnitTestFramework\\UnitTestFramework.js");
+
+function testModuleScript(source, message, shouldFail) {
+    let testfunc = () => WScript.LoadModule(source);
+
+    if (shouldFail) {
+        let caught = false;
+
+        // We can't use assert.throws here because the SyntaxError used to construct the thrown error
+        // is from a different context so it won't be strictly equal to our SyntaxError.
+        try {
+            testfunc();
+        } catch(e) {
+            caught = true;
+
+            // Compare toString output of SyntaxError and other context SyntaxError constructor.
+            assert.areEqual(e.constructor.toString(), SyntaxError.toString(), message);
+        }
+
+        assert.isTrue(caught, `Expected error not thrown: ${message}`);
+    } else {
+        assert.doesNotThrow(testfunc, message);
+    }
+}
+
+var tests = [
+    {
+        name: "Basic import namespace",
+        body: function () {
+           testModuleScript('import * as foo from "ValidExportStatements.js"; for (var i in foo) helpers.writeln(i + "=" + foo[i]);', '', false);
+       }
+    },
+    // {
+        // name: "import namespace with verification",
+        // body: function () {
+           // // testModuleScript('import * as foo from "moduleExport1.js"; for (var i in foo) WScript.Echo(i + "=" + foo[i]);', '', false);
+           // // testModuleScript('import * as foo from "moduleExport1.js"; foo.verifyNamespace(foo);', '', false);
+           // // testModuleScript('import * as foo1 from "moduleExport1.js"; foo1.changeContent(); foo1.verifyNamespace(foo1);', '', false);
+        // }
+    // },
+    {

```

Comment:
This is disabled due to the module export symbol AV? Should be fixed by #1384 fyi